recipes/sdl/all/conanfile.py

```diff
@@ -379,6 +391,9 @@ def package_info(self):
 
         postfix = "d" if self.settings.os != "Android" and self.settings.build_type == "Debug" else ""
 
+        if self.version >= "2.0.24" and is_msvc(self) and not self.options.shared:
```

Comment:
Don't you need to use version helpers to compare?
```suggestion
        if tools.Version(self.version) >= "2.0.24" and is_msvc(self) and not self.options.shared:
```

Quite frankly, I don't understand why this line does not fail.